reconcile/slack_usergroups.py

```diff
@@ -689,24 +663,40 @@ def act(
                 usergroup, State()
             )
 
+            slack_client = slack_map[workspace].slack
+
+            desired_ug_state.users = {
+                SlackObject(pk=pk, name=name)
+                for pk, name in slack_client.get_users_by_names(
+                    sorted(desired_ug_state.user_names)
+                ).items()
+            }
+
+            desired_ug_state.channels = {
```

Comment:
similar here, `desired_ug_state.channels` only used in `_update_usergroup_from_state`.